clang/lib/Driver/ToolChains/ClassicFlang.cpp

```diff
@@ -955,7 +955,7 @@ void ClassicFlang::ConstructJob(Compilation &C, const JobAction &JA,
   LowerCmdArgs.push_back("-x"); LowerCmdArgs.push_back("121"); LowerCmdArgs.push_back("0x800");
   LowerCmdArgs.push_back("-x"); LowerCmdArgs.push_back("54"); LowerCmdArgs.push_back("0x10");
   LowerCmdArgs.push_back("-x"); LowerCmdArgs.push_back("70"); LowerCmdArgs.push_back("0x40000000");
-  LowerCmdArgs.push_back("-x"); LowerCmdArgs.push_back("249"); LowerCmdArgs.push_back("90"); // LLVM version
+  LowerCmdArgs.push_back("-x"); LowerCmdArgs.push_back("249"); LowerCmdArgs.push_back("11"); // LLVM version
```

Comment:
Shouldn't this be changed to `"110"` instead of `"11"`? Unless you are also changing the way it is used in flang.

Reply:
Thanks for catching this :) While moving back & forth with clang & flang I forgot the convention.
All builds are clean (locally)  https://github.com/flang-compiler/flang/pull/898 this should also pass after this change